### PR TITLE
Update readOwnership by changing Readuser to ReadOwner model.

### PR DESCRIPTION
**Pull request by varhanovdm**
dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009/issues/ita-social-projects/home/333)

## Code reviewers

- [ ] @bbogdasha 
- [ ] @kh0ma 
- [ ] @Pavlo-Ponomarenko 
- [ ] @DzigMS 

## Summary of issue

Change field 'user' to field 'owner' in ReadOwnership model.

## Summary of change

Model ReadOwnership was successfully updated by changing ReadUser to ReadOwner and removing the email field.

## Testing approach

Not involved.

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: home-application/src/main/java/com/softserveinc/ita/homeproject/application/mapper/config/impl/ReadOwnerMappingConfig.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@Component
+@RequiredArgsConstructor
+public class ReadOwnerMappingConfig implements HomeMappingConfig<OwnershipDto, ReadOwnership> {
+
+    @Override
+    public void addMappings(TypeMap<OwnershipDto, ReadOwnership> typeMap) {
+        typeMap.addMapping(OwnershipDto::getUser, ReadOwnership::setOwner);
+    }
+}
```
Comment (kh0ma) [inline home-application/src/main/java/com/softserveinc/ita/homeproject/application/mapper/config/impl/ReadOwnerMappingConfig.java]: Please add newline to the end of the file https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

File: home-open-api/src/main/resources/yaml/models/owner.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      type: array
+      example: [
+        { "id": 1,
+          "type": "email",
+          "main": false,
+          "email": "readUserEmail@example.com"
+        }
+      ]
+      items:
+        $ref: 'contact.yaml#/ReadContact'
```
Comment (kh0ma) [inline home-open-api/src/main/resources/yaml/models/owner.yaml]: Newline

**Review by kh0ma — CHANGES_REQUESTED** [review]

File: home-application/src/main/java/com/softserveinc/ita/homeproject/application/mapper/config/impl/ReadOwnerMappingConfig.java (hunk 1 above)
Comment (varhanovdm) [reply home-application/src/main/java/com/softserveinc/ita/homeproject/application/mapper/config/impl/ReadOwnerMappingConfig.java]: Ok

**Review by bbogdasha — APPROVED** [review]

**Review by Pavlo-Ponomarenko — APPROVED** [review]

File: home-open-api/src/main/resources/yaml/models/owner.yaml (hunk 2 above)
Comment (varhanovdm) [reply home-open-api/src/main/resources/yaml/models/owner.yaml]: Fixed

**Review by kh0ma — APPROVED** [review]

**Pull request merged**